### PR TITLE
DMTCP configuration file docs

### PR DESCRIPTION
**Pull request by ikaneshiro**
Signed-off-by: Ian Kaneshiro <iankane@umich.edu>

Related to https://github.com/apptainer/apptainer-userdocs/pull/63

### CODE REVIEW

File: configfiles.rst [hunk 1]
```diff
@@ -1119,3 +1119,55 @@ of key servers used to verify container signatures by default.
...
 <{userdocs}/endpoint.html>`_.
+
+
+.. _dmtcp_configuration:
+
+*****************
+ dmtcp-conf.yaml
+*****************
+
+The ``dmtcp-conf.yaml`` is a YAML configuration file is used to specify the
```
Comment (DrDaveD) [inline configfiles.rst:1130]: Insert "that" after "file"

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Should there also be a link to this somewhere else in the admin docs?

**Comment by ikaneshiro** [comment]
I don't think it needs to be linked elsewhere since it will show up in the `Configuration Files` drop down menu

**Comment by ikaneshiro** [comment]
Nevermind, I see what you are saying. Ill update `index.rst`

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**